homeassistant/components/growatt_server/sensor/total.py

```diff
@@ -26,13 +26,15 @@
         api_key="todayEnergy",
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
     ),
     GrowattSensorEntityDescription(
         key="total_output_power",
         translation_key="total_output_power",
         api_key="invTodayPpv",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
This is wrong, see https://github.com/home-assistant/core/blob/bb7dc079ce6160ebdd9828864ac9f8df255083b7/homeassistant/components/sensor/const.py#L637